src/compareSnapshots.js

```diff
@@ -11,27 +13,36 @@ function makeAbsolute(url, endpoint) {
 function imageDiff({ bitmap1, bitmap2, compareThreshold }) {
   const len = bitmap1.width * bitmap1.height * 4;
   for (let i = 0; i < len; i += 4) {
-    const distance =
-      colorDelta(
-        [
-          bitmap1.data[i],
-          bitmap1.data[i + 1],
-          bitmap1.data[i + 2],
-          bitmap1.data[i + 3],
-        ],
-        [
-          bitmap2.data[i],
-          bitmap2.data[i + 1],
-          bitmap2.data[i + 2],
-          bitmap2.data[i + 3],
-        ],
-      );
+    const distance = colorDelta(
+      [
+        bitmap1.data[i],
+        bitmap1.data[i + 1],
+        bitmap1.data[i + 2],
+        bitmap1.data[i + 3],
+      ],
+      [
+        bitmap2.data[i],
+        bitmap2.data[i + 1],
+        bitmap2.data[i + 2],
+        bitmap2.data[i + 3],
+      ],
+    );
     if (distance > compareThreshold) {
       return distance;
     }
   }
 }
 
+async function fetchPngWithRetry(url) {
+  const bitmap = await asyncRetry(() => fetchPng(url), {
+    retries: 3,
```

Comment:
Maybe increase this to 5?
```suggestion
    retries: 5,
```

The cost of retrying is pretty low and the cost of a failed job is pretty high.